plugin/src/App/StatusPages/Settings/Setting.lua

```diff
@@ -120,37 +121,55 @@ function Setting:render()
 				Size = UDim2.new(1, 0, 1, 0),
 				BackgroundTransparency = 1,
 			}, {
-				Name = e("TextLabel", {
-					Text = (
-						if self.props.experimental
-							then '<font color="#FF8E3C">⚠ </font>'
-							elseif
-								self.props.developerDebug
-							then '<font family="rbxasset://fonts/families/Guru.json" color="#35B5FF">⚑ </font>' -- Guru is the only font with the flag emoji
-							else ""
-					) .. self.props.name,
-					Font = Enum.Font.GothamBold,
-					TextSize = 17,
-					TextColor3 = theme.Setting.NameColor,
-					TextXAlignment = Enum.TextXAlignment.Left,
-					TextTransparency = self.props.transparency,
-					RichText = true,
-
-					Size = UDim2.new(1, 0, 0, 17),
-
-					LayoutOrder = 1,
+				Heading = e("Frame", {
+					Size = UDim2.new(1, 0, 0, 16),
 					BackgroundTransparency = 1,
+				}, {
+					Layout = e("UIListLayout", {
+						VerticalAlignment = Enum.VerticalAlignment.Center,
+						FillDirection = Enum.FillDirection.Horizontal,
+						SortOrder = Enum.SortOrder.LayoutOrder,
+						Padding = UDim.new(0, 5),
+					}),
+					Tag = if self.props.unstable or self.props.debug
+						then e(Tag, {
+							layoutOrder = 1,
+							transparency = self.props.transparency,
+							text = if self.props.unstable then "UNSTABLE" else "DEBUG",
+							icon = if self.props.unstable
+								then Assets.Images.Icons.Warning
+								else Assets.Images.Icons.Debug,
+							color = if self.props.unstable
+								then settingsTheme.Setting.UnstableColor
+								elseif self.props.debug then settingsTheme.Setting.DebugColor
+								else settingsTheme.BrandColor,
+						})
+						else nil,
```

Comment:
This section makes me kind of sad but since there's only two types of tags it's probably fine. If we ever end up adding another tag, I'm gonna propose we generalize it somehow though.

Reply:
Nah you're right, this is lazy. I fixed it, thanks for calling it out.